### PR TITLE
Web development course capstone project Pull request.

### PR DESCRIPTION
**Pull request by HERMON-1995**
#Hello, there 😃 

> In this capstone project is all about building Web Development online course website using HTML, CSS and JavaScript based on an online website for a conference designed by [Cindy Shin in Behance](https://www.behance.net/adagio07).

> I build only these 2 pages:

     1. The home page.
     2. The about page.

> Each of these pages have versions for 2 different screen sizes:

      - [Mobile: up to 768px wide] 
      - [Desktop: 768px or wider] 

***Interactions***

   **Links**

      - The home page has a link in the menu to the about page.
      - The logo in the header links to the home page.

   **Mobile menu**

     - When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
     -  The mobile menu has a close (X) button that closes the menu.

   **Dynamic page**

     - The section "Featured speakers" is created dynamically in JavaScript.
     - I used a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this 
        section dynamically.

> Here is my short video describing about the project **[https://www.loom.com/share/ff66d7db63b34304a7ebaf77f721cdc3]**.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,579 @@
...
+}
+
+.left-cont h3 {
+  font-size: 25px;
+}
+
+.right-cont p {
+  padding-top: 10px;
+  padding-left: 25px;
+}
```
Comment (leonardodiasb) [inline style.css]: - [ ] To make your footer style looks consistent and more professional, please use the same sizing and dimensions used for both footers. The footer for the about page has smaller font sizes and looks more like the template used. Kindly reduce the size to match the template.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        <img class="git-img" src="./images/git-last-img.png" alt="git image">
+        <h3>Ultimate Git course</h3>
+        <p>Everything you need to know to use Git & GitHub,work effectively as a team. </p>
+    </div>
+    <div class="course">
+        <img class="redux-img" src="./images/redux-icone.png" alt="redux image">
+        <h3>Ultimate Redux course</h3>
+        <p>Go from beginner to expert in a couple of hours in this short course redux.</p>
+    </div>
+  </div>
```
Comment (leonardodiasb) [inline index.html]: - [ ] Your articles section is good and very similar to the template, but to make the HTML code more semantically correct, please use the `article` tag to wrap each article.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @HERMON-1995, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Good video presentation. ✔️
 - Professional README and PR description. ✔️ 
 - No linter errors. ✔️ 
 - Project correctly deployed to GitHub Pages. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.
- [ ] I can see that you added the 'Lato' font-family to some text in your pages, but other texts do not have the same font, making the visual of your pages not consistent and not a professional look. I recommend you use the 'Lato' font-family as a base font-family for your entire application to match the template font-families used. Please add the 'Lato' font family for every text in your application.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @HERMON-1995    :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Highlights
- Professional README file :heavy_check_mark: 
- Professional PR description :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (HERMON-1995) [reply index.html]: Thank you for the feedback. I appreciate your suggestion to use the `<article>` tag for each article to improve the semantic correctness of my HTML code. I will make the necessary changes. 😁

File: style.css (hunk 1 above)
Comment (HERMON-1995) [reply style.css]: Thank you for your feedback on my footer style. I will make the necessary adjustments to ensure consistency and professionalism. 😄

**Comment by HERMON-1995** [comment]
Thank you for approving my project. I am grateful for the opportunity to work on this exciting initiative. 😊

**Pull request merged**